client/main/notifications.js

```diff
@@ -1,19 +1,36 @@
 
 'use strict';
 
-// const Notification = require('./notification');
+var $ = require('jquery');
+
+const NotificationView = require('./notification');
 
 const Notifications = function($el) {
     this.$el = $el;
-    this._requests = [ ];
+    this._list = [ ];
 };
 
 Notifications.prototype.notify = function(request) {
 
     console.log('notification received');
     console.log(request.attributes);
+    
+    var $note = $('<div id="notification"></div>');
+    this.$el.append($note);
+    request.set("index", this._list.length);
+    request.parent = this;
+    var note = new NotificationView({el : $note, model : request});
+    this._list.push({request: request, view: note});
+};
+
+Notifications.prototype.remove = function(view) {
 
-    this._requests.push(request);
+    var index = this._list.map(function(e) { return e.view; }).indexOf(view);
```

Comment:
arrow functions bro! :)

    var index = this._list.map(e => e.view).indexOf(view);